bitnami/moodle/values.yaml

```diff
@@ -227,7 +227,7 @@ containerPorts:
 ## For minikube, set this to NodePort, for ingress ClusterIP, elsewhere use LoadBalancer
 ##
 service:
-  type: LoadBalancer
+  type: NodePort
```

Comment:
Please don't change the default values.

Reply:
this was an accident when I was doing some testing.